doc/engineering-reference/src/building-system-simulation-system-manager/zone-equipment-simulation.tex

```diff
@@ -111,7 +111,7 @@ \subsection{Simulation}\label{simulation-003}
 
 For zone equipment components that are part of the demand side of an air loop the simulation sequence is effectively performed in the direction of the air flow. First the supply air plenums and zone splitters are simulated in their input order. Then the air terminal units are simulated followed by the zone return plenums and zone mixer. Each air terminal unit sets its inlet node to the air mass flow rate required to satisfy its zone load. These mass flow rates are then passed back upstream to the air loop demand-side inlet nodes at the end of each zone equipment simulation sequence. These demand-side inlet mass flow rates are then used as mass flow rate setpoints in the air loop supply-side simulation.
 
-If multiple air-conditioning components are attached to a zone, the components are simulated in the order specified by the user assigned priority given in the ZoneHVAC:EquipmentList object.
+If multiple air-conditioning components are attached to a zone, the components are simulated in the order specified by the user assigned priority given in the ZoneHVAC:EquipmentList object, using one of four load distribution schemes. (See Zone Equipment Load Distribution below.)
```

Comment:
Could say: `See Zone Equipment Load Distribution described in \ref{zone-equipment-load-distribution}`  :smile:

Reply:
Done, and the link actually works!